website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -89,13 +89,13 @@ By default, the footer displays below the options list and contains content and/
 
 ##### Single
 
-The footer for the Single component includes a total count, which is customizable to fit the context.
+The footer for the Single component includes a total count, which is customizable to fit the context. However, we recommend using "{xx} total" to maintain consistency across products.
 
 ![Generic content enabled before the list in both Single and Multiple components](/assets/components/form/super-select/super-select-single-variant-footer-with-counter.png)
 
 ##### Multiple
 
-The footer for the Multiple component includes buttons to modify the options and a count of the number of options selected out of the total.
+The footer for the Multiple component includes a selected count and buttons to modify the options. While the content can be customized, we recommend using "{xx} selected of {xx} total" to drive consistency across products, similar to the Single component.
```

Comment:
```suggestion
The footer for the Multiple component includes a selected count and list modifier buttons. While the count text can be customized, we recommend using "{xx} selected of {xx} total" to drive consistency across products, similar to the Single component.
```
(non-blocking, nitpick) I might add this just to clarify a teeny bit. Otherwise this looks good to me!